src/app/store/actions/product.actions.ts

```diff
@@ -10,8 +10,8 @@ export const GetProductListSuccess = createAction(
   props<{ products: Product[] }>()
 );
 
-export const GetProductListeError = createAction(
-  '[Product] Error in reading the list of products',
+export const Error = createAction(
```

Comment:
One small nitpick would be to name these into something less generic, i.e., ProductError so that when you're importing them you'll have an easier time